features/bevragen/zoek-met-geslachtsnaam-voornamen-en-gemeente-van-inschrijving/fout-cases.feature

```diff
@@ -0,0 +1,378 @@
+    | instance | /haalcentraal/api/brp/personen                              |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name      | reason                  |
+    | required | voornamen | Parameter is verplicht. |
+
+  @fout-case
+  Scenario: Alleen de gemeenteVanInschrijving parameter is niet opgegeven
+    Als personen wordt gezocht met de volgende parameters
+    | naam          | waarde                               |
+    | type          | ZoekMetNaamEnGemeenteVanInschrijving |
+    | voornamen     | Jan                                  |
+    | geslachtsnaam | Jansen                               |
+    | fields        | burgerservicenummer                  |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                      |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.   |
+    | status   | 400                                                         |
+    | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving.     |
+    | code     | paramsCombination                                           |
+    | instance | /haalcentraal/api/brp/personen                              |
+    En heeft het object de volgende 'invalidParams' gegevens
+    | code     | name                    | reason                  |
+    | required | gemeenteVanInschrijving | Parameter is verplicht. |
+
+  @fout-case
+  Scenario: Een lege string is opgegeven als voornamen, geslachtsnaam en gemeente van inschrijving waarde
+    Als personen wordt gezocht met de volgende parameters
+    | naam                    | waarde                               |
+    | type                    | ZoekMetNaamEnGemeenteVanInschrijving |
+    | voornamen               |                                      |
+    | geslachtsnaam           |                                      |
+    | gemeenteVanInschrijving |                                      |
+    | fields                  | burgerservicenummer                  |
+    Dan heeft de response een object met de volgende gegevens
+    | naam     | waarde                                                                            |
+    | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1                       |
+    | title    | Minimale combinatie van parameters moet worden opgegeven.                         |
+    | status   | 400                                                                               |
+    | detail   | De foutieve parameter(s) zijn: gemeenteVanInschrijving, geslachtsnaam, voornamen. |
+    | code     | paramsCombination                                                                 |
+    | instance | /haalcentraal/api/brp/personen                                                    |
+    En heeft het object de volgende 'invalidParams' gegevens
```

Comment:
Is de waarde 'Parameter is verplicht.' voor reason logisch in deze situatie. In principe worden de parameters geslachtsnaam en geboortedatum immers meegegeven maar met een verkeerde waarde.
De reasons 'Waarde is geen geldige datum.' en 'Waarde voldoet niet aan patroon ^[a-zA-Z0-9À-ž .-']{1,200}$|^[a-zA-Z0-9À-ž .-']{3,199}*{1}$.' lijken mij logischer.